src/response.js

```diff
@@ -0,0 +1,11 @@
+export class Response {
+  constructor(code = 200, body = {}, headers = {}) {
+    this.code = code;
+    this.body = body;
+    this.headers = headers;
+  }
+
+  get isErrored() {
+    return this.code >= 400 ? true : false;
```

Comment:
Thinking about it, this is fine since it's only for error... Should we also have method just for success?